tests/acceptance/features/apiOcm/share.feature

```diff
@@ -288,3 +288,139 @@ Feature: an user shares resources usin ScienceMesh application
       | Space Viewer     |
       | Space Editor     |
       | Manager          |
+
+  @issue-9908
+  Scenario: sharer lists the shares shared by him to a federated user
+    Given using server "LOCAL"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+    And user "Alice" has uploaded file with content "ocm test" to "/textfile.txt"
```

Comment:
IMHO, placing the step:
` And user "Alice" has uploaded file with content "ocm test" to "/textfile.txt"` just below the `Given step` sounds more better